packages/webamp/js/webampLazy.tsx

```diff
@@ -505,13 +507,17 @@ class Webamp {
       onMount = resolve;
     });
 
+    if (contained && getComputedStyle(node)?.position === "static") {
+      node.style.position = "relative";
```

Comment:
What if instead of silently modifying the DOM node we threw and error informing the user that they should avoid using `position: static` for the element into which Webamp is rendered?